Solutions/Digital Shadows/Data Connectors/Digital Shadows/digitalshadowsARM.json

```diff
@@ -0,0 +1,253 @@
+                "[resourceId('Microsoft.Insights/components', variables('FunctionName1'))]"
+            ],
+            "kind": "functionapp,linux",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "name": "[variables('FunctionName1')]",
+                "httpsOnly": true,
+                "clientAffinityEnabled": true,
+                "alwaysOn": true,
+                "reserved": true,
+                "siteConfig": {
+                    "linuxFxVersion": "python|3.8"
+                }
+            },
+            "resources": [
+                {
+                    "apiVersion": "2018-11-01",
+                    "type": "config",
+                    "name": "appsettings",
+                    "dependsOn": [
+                        "[concat('Microsoft.Web/sites/', variables('FunctionName1'))]"
+                    ],
+                    "properties": {
+                        "FUNCTION_NAME": "[variables('FunctionName1')]",
+                        "FUNCTIONS_EXTENSION_VERSION": "~3",
+                        "FUNCTIONS_WORKER_RUNTIME": "python",
+                        "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('Microsoft.insights/components', variables('FunctionName1')), '2015-05-01').InstrumentationKey]",
+                        "APPLICATIONINSIGHTS_CONNECTION_STRING": "[reference(resourceId('microsoft.insights/components', variables('FunctionName1')), '2015-05-01').ConnectionString]",
+                        "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('ConnectorName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('ConnectorName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                        "logAnalyticsUri": "[variables('LogAnaltyicsUri')]",
+                        "HighVariabilityClassifications": "[parameters('HighVariabilityClassifications')]",
+                        "ClassificationFilterOperation": "include",
+                        "WorkspaceID": "[parameters('WorkspaceID')]",
+                        "WorkspaceKey": "[parameters('WorkspaceKey')]",
+                        "DigitalShadowsAccountID": "[parameters('DigitalShadowsAccountID')]",
+                        "HistoricalDays": "[parameters('HistoricalDays')]",
+                        "DigitalShadowsKey": "[parameters('DigitalShadowsKey')]",
+                        "DigitalShadowsSecret": "[parameters('DigitalShadowsSecret')]",
+                        "DigitalShadowsURL": "[parameters('DigitalShadowsURL')]",
+                        "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/digitalshadows/Azure-Sentinel/raw/DigitalShadows-connector-v1/Solutions/Digital%20Shadows/Data%20Connectors/Digital%20Shadows/digitalshadowsConnector.zip"

```

Comment:
Hey @shikhin-metron, WEBSITE_RUN_FROM_PACKAGE with this link (https://aka.ms/sentinel-DigitalShadow-functionapp), thanks.